### PR TITLE
Added test case with Ad Controller

### PR DESCRIPTION
**Pull request by RomanTysiachnik**
## Added test case with Ad Controller
This test case includes two test cases:
- Case with the Ad loading from the beginning;
- Case with the Ad playing via AirPlay.

### CODE REVIEW

File: SnapshotTests/Carthage/Checkouts/ios-snapshot-test-case/build.sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+set -eu
+
+function ci_lib() {
```
Comment (AndriiMoskvin) [inline SnapshotTests/Carthage/Checkouts/ios-snapshot-test-case/build.sh]: Why not fastlane?

File: SnapshotTests/DefaulController/DefaulController.swift [hunk 2]
```diff
@@ -0,0 +1,15 @@
+//  Copyright © 2017 rtysiachnik. All rights reserved.
```
Comment (AndriiMoskvin) [inline SnapshotTests/DefaulController/DefaulController.swift]: Change everywhere in this PR to - `//  Copyright © 2017 Oath. All rights reserved.`

File: SnapshotTests/Carthage/Checkouts/ios-snapshot-test-case/LICENSE [hunk 3]
```diff
@@ -0,0 +1,29 @@
+BSD License
```
Comment (AndriiMoskvin) [inline SnapshotTests/Carthage/Checkouts/ios-snapshot-test-case/LICENSE]: Is it required to have this checked-in in our sources?

**Comment by AndriiMoskvin** [comment]
Can this PR be split into several? It has a lot of changes...

File: SnapshotTests/DefaulController/DefaulController.swift (hunk 2 above)
Comment (RomanTysiachnik) [reply SnapshotTests/DefaulController/DefaulController.swift]: Got it.

File: SnapshotTests/Carthage/Checkouts/ios-snapshot-test-case/LICENSE (hunk 3 above)
Comment (RomanTysiachnik) [reply SnapshotTests/Carthage/Checkouts/ios-snapshot-test-case/LICENSE]: Do we actually need Checkouts at all in our project? Is it a trash or there is a point with a license for example and so on.

**Comment by RomanTysiachnik** [comment]
@AndriiMoskvin we've already split Snapshot Testing PR. The main one contains a new library and Fastlane integration. Right after this PR will be merged, all PRs with test cases will have changes only with new test cases.

**Comment by RomanTysiachnik** [comment]
@AndriiMoskvin ready for review

**Pull request merged**